src/modules/learning-object-service/collections/CollectionController.ts

```diff
@@ -1,12 +1,137 @@
+
+    /**
+     * @swagger
+     * /{collection}/metrics:
+     *  get:
+     *    description: Gets a collection's metrics
+     *    tags:
+     *      - Learning Object Service
+     *    parameters:
+     *      - in: path
+     *        name: collection
+     *        schema:
+     *            type: string
+     *        required: true
+     *        description: The name of the collection
+     *    responses:
+     *      200:
+     *        description: OK
+     *        content:
+     *          application/json:
+     *            schema:
+     *              $ref: '#/components/schemas/CollectionMetricBody'
+     *      500:
+     *        description: INTERNAL SERVICE ERROR - Collection not found
+     */
+    router.route('/:collection/metrics').get(this.proxyRequest((req: Request) => `/${encodeURIComponent(req.params.collection)}/metrics`));
+    
+    /**
+     * @swagger
+     * /collections/{name}/meta:
+     *  get:
+     *    description: Gets a collection's meta data
+     *    tags:
+     *      - Learning Object Service
+     *    parameters:
+     *      - in: path
+     *        name: name
+     *        schema:
+     *            type: string
+     *        required: true
+     *        description: The name of the collection
+     *    responses:
+     *      200:
+     *        description: OK
+     *        content:
+     *          application/json:
+     *            schema:
+     *              $ref: '#/components/schemas/Collection'
+     *      404:
+     *        description: NOT FOUND - Collection not found
+     */
+    router.get('/collections/:name/meta', this.proxyRequest((req: Request) => `/collections/${encodeURIComponent(req.params.name)}/meta`));
+    
+    /**
+     * @swagger
+     * /collections:
+     *  get:
+     *    description: Gets a list of collections
+     *    tags:
+     *      - Learning Object Service
+     *    responses:
+     *      200:
+     *        description: OK
+     *        content:
+     *          application/json:
+     *            schema:
+     *              type: array
+     *              items:
+     *                properties:
+     *                  name:
+     *                    type: string
+     *                    example: nccp
+     *                    required: true
+     *                    description: The collection name
+     *                  abvName:
+     *                    type: string
+     *                    example: nccp
+     *                    required: true
+     *                    description: The collection's abbreviated name
+     *                  hasLogo:
+     *                    type: boolean
+     *                    example: true
+     *                    required: true
+     *                    description: True if the collection has a logo
+     */
+    router.get('/collections', this.proxyRequest((req: Request) => LEARNING_OBJECT_ROUTES.GET_COLLECTIONS));
+
+    /**
+     * @swagger
```

Comment:
This is just a comment, but I wonder if this route is used anymore since I believe this gets handled by the submission pipeline. I could be wrong but just wanted to put it down here.

Reply:
I'll write a note in the comment but I do see that you commented that the route was deprecated in the client file.  I would rather it removed from the client first then go back and update gateway then the other way around.

Reply:
I want to check it in the service. If it's not used let's just purge it from the system all at once